test/plugin/base_test.rb

```diff
@@ -2374,7 +2506,12 @@ def verify_cascading_json_detection_with_log_entry_fields(
         d.emit(input_log_entry)
         d.run
       end
-      verify_log_entries(1, COMPUTE_PARAMS, 'jsonPayload') do |entry|
+      expected_params = COMPUTE_PARAMS.dup
+      if log_entry_field == 'labels'
+        expected_value = expected_value.merge(COMPUTE_PARAMS[:labels])
```

Comment:
Are you sure this `merge` is in the right direction? This would make the label values from `COMPUTE_PARAMS[:labels]` override the ones in `expected_value`...

Reply:
The test data has no overlap, so it should not matter. But I changed it anyway to match the actual precedence.